scripts/tests/harness.sh

```diff
@@ -47,6 +47,21 @@ function forest_run_node_detached {
   $FOREST_PATH --chain calibnet --encrypt-keystore false --log-dir "$LOG_DIRECTORY" --detach --save-token ./admin_token --track-peak-rss
 }
 
+function forest_run_node_stateless_detached {
+  CONFIG_PATH="./stateless_forest_config.toml"
+  echo "${CONFIG_PATH}"
+  echo "Running forest in stateless and detached mode"
+  {
+    echo "[client]"
+    echo "data_dir = \"/tmp/stateless_forest_data\""
+    echo ""
+    echo "[network]"
+    echo "listening_multiaddrs = [\"/ip4/127.0.0.1/tcp/0\"]"
+  } > $CONFIG_PATH
```

Comment:
Do you think HEREDOC would be more readable and easier to modify later?

Reply:
That's good idea, fixed.